tests/pytorch/test_torch_save_load.py

```diff
@@ -168,6 +168,15 @@ def test_fp8_model_checkpoint(
         fp8_meta_bwd.scale.copy_(fp8_meta_bwd_ref["scale"])
         fp8_meta_bwd.scale_inv.copy_(fp8_meta_bwd_ref["scale_inv"])
         del fp8_meta_fwd, fp8_meta_bwd
+    
+    # [ This is used to test the scenario where save_fp8_model is set to False and load_fp8_model is set to True ]
```

Comment:
"This is part of logic that tests `save_fp8_model=False` and `load_fp8_model=True`"